### PR TITLE
refactor: Return Torrent when client.seed finds a duplicate

### PR DESCRIPTION
**Pull request by Bruce-Hopkins**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[X] Bug fix
[ ] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
When calling `client.seed` it would usually throw an error. Now it passes the torrent instance in the callback and adds a warning on the console.

**Which issue (if any) does this pull request address?**
 #2215

**Is there anything you'd like reviewers to focus on?**

The way I added the callback, there is no way to know if the torrent is a duplicate or not. Should we also pass a boolean to represent whether the torrent is a duplicate or not?

### CODE REVIEW

**Comment by DiegoRBaquero** [comment]
Could you create a test case?

**Comment by Bruce-Hopkins** [comment]
By test case, do you mean unit tests or example code?

**Comment by RangerMauve** [comment]
> By test case, do you mean unit tests or example code?

Probably unit tests 😁

**Comment by Bruce-Hopkins** [comment]
I added tests to show that the torrents are duplicates if they seed the same file.  I don't know if there are any standards to the tests, let me know if they are wrong.

**Comment by Bruce-Hopkins** [comment]
I'm new to contributing. What is CodeQL and why is it not allowing the merge? The issue it's having isn't a file I changed.

File: index.js [hunk 1]
```diff
@@ -393,7 +393,9 @@ class WebTorrent extends EventEmitter {
 
           const existingTorrent = this.get(torrentBuf)
           if (existingTorrent) {
-            torrent._destroy(new Error(`Cannot add duplicate torrent ${existingTorrent.infoHash}`))
+            console.warn('A torrent with the same seed is already in the client.')
```
Comment (DiegoRBaquero) [inline index.js:396]: Thoughts on the wording?

```suggestion
            console.warn('A torrent with the same id is already being seeded')
```

**Comment by Bruce-Hopkins** [comment]
@DiegoRBaquero I agree with the change. I just added it.

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**